src/Inkluzitron/Modules/UserModule.cs

```diff
@@ -102,16 +104,43 @@ public async Task ShowUserInfoAsync([Name("uživatel")] IUser user = null)
 
             var guildUser = await UsersService.GetUserFromHomeGuild(user);
             if (guildUser != null)
-                descBuilder.Append("**Na serveru od:** ").AppendLine(guildUser.JoinedAt.Value.LocalDateTime.ToShortDateString());
+                descBuilder.AppendLine($"**V Bobánkově od:** {guildUser.JoinedAt.Value.LocalDateTime:d. M. yyyy}");
             else
-                descBuilder.AppendLine("Již není na serveru");
+                descBuilder.AppendLine("Již není v Bobánkově");
+
+            var inviteUsed = await DbContext.Invites.AsQueryable()
+                .Where(i => i.UsedByUserId == user.Id)
+                .OrderByDescending(i => i.GeneratedAt)
+                .Select(i => i.GeneratedByUserId)
+                .FirstOrDefaultAsync();
+
+            var invitedBy = await UsersService.GetDisplayNameAsync(inviteUsed);
+
+            if (invitedBy != null)
+                descBuilder.AppendLine($"**Pozván od:** {invitedBy}");
 
             var embed = new EmbedBuilder()
                 .WithAuthor(user)
                 .WithCurrentTimestamp()
                 .WithDescription(descBuilder.ToString())
                 .WithFooter("informace o uživateli");
 
+            var invitedUsersIds = DbContext.Invites.AsQueryable()
+                    .Where(i => i.GeneratedByUserId == user.Id && i.UsedByUserId.HasValue)
```

Comment:
A tady taky ne. To se bot nad naší obří databázi chudák pořádně zapotí.

Reply:
Bot to zvládne 😄 Já mu věřím